routing/RouteSolver.py

```diff
@@ -0,0 +1,134 @@
+##   Program: Basic Route Solver for Factory Roaming Robot                      ##
+##   Author: Collin Duddy                                                       ##
+##   Class: EECS_542 Senior Design University of Kansas                         ##
+##                                                                              ##
+##   Description: This program will take in user specified input of a graph,    ##
+##                or map of nodes. It will then calculate all routes possible   ##
+##                from the users start node to the users end-node. This data    ##
+##                will be piped to a Raspberry Pi that will then send the       ##
+##                data to an arduino for roaming the map.                       ##
+##################################################################################
+import Graph
+import sys
+import re
+##################################################################################
+##                                                                              ##
+##                            Global Variables                                  ##
+##                                                                              ##
+##################################################################################
+graph = Graph.Graph()
+##################################################################################
+##                                                                              ##
+##                            Functions/Methods                                 ##
+##                                                                              ##
+##################################################################################
+
+def is_int(s):
+    try:
+        int(s)
+        return True
+    except ValueError:
+        return False
+
+def loadConfig(configFile):
+    f = open(configFile,"r")
+    lines = f.readlines()
+    currNode = ""
+    bad_chars = '(){}<>[]'
+    for i in range(0,len(lines)):
+        line= lines[i]
+        if "Next Node:" in line:
+            graph.add_vertex(lines[i+1].strip())
+            currNode = lines[i+1].strip()
+        elif "Connected Nodes:" in line:
+            connectedNodes = lines[i+1].split(';')
+            for node in connectedNodes:
+                nodeName = node.split(',')
+                for c in bad_chars: nodeName[0] = nodeName[0].replace(c, "")
+                for c in bad_chars: nodeName[1] = nodeName[1].replace(c, "")
+                nodeName[1].rstrip()
+                #graph.add_edge((currNode,(nodeName[0],nodeName[1])))
+                # Adding directions is causing an issue. May have to come up with
+                #another solution for direction
+                graph.add_edge((currNode,nodeName[0]))
+
+def getGraphDetailsFromUser():
+    numberOfNodes = 0;
+    while True:
+        numberOfNodes = input("How many nodes are in your graph? ")
```

Comment:
Instead of `is_int` do the following

```python
while True:
    try:
        node_count = int(input('How many nodes are in your graph?'))
    except ValueError:
        print('Invalid input. Please input an integer')
        continue
```